### ISSUE TITLE
Shattered Halls [normal] Last boss and stuck in combat long times.

### ISSUE THREAD

**Issue by t0mpp44** [ccshiro/cc-buglist#193]
So we did a full clear of shattered halls normal and after the event (at around half way into the instance) we found ourselves stuck in combat even tho all mobs were killed. This lasted all the way to last boss and even after we had killed the boss. Vanishing or Feign deathing cleared the combat. 

Also in the last bossfight. The boss should be charging from player to player but instead it just ran around the room with very high speed.

**Comment by FurioCC** [comment]
You are wrong about Kargath Bladefist behaviour. His Blade dance is random bouncing around the room. That's why corner stacking tactic worked best on retail.

**Comment by descendance** [comment]
Here is a video of the boss encounter from before patch 2.1.0.
At around 30 sec marks you will see him blade dancing around randomly.

https://www.youtube.com/watch?v=F1VuguUUcjI&index=44&list=UUUjuRH2D-Krf9nC46fH7luA

**Comment by vauhs** [comment]
We had a similar issue in H-Shattered Halls with the combat bug. Hunter could FD and mage could use invis to get out of combat, but once they got into combat again it would do the same. I figured out it was the pet that was doing it from the hunter. Hunter dismiss'd pet/resummoned, everyone alt f4'd to get out of combat and didn't have any issues after that. We tried alt f4ing before we dismiss'd the pet but still got combat bug. So I think it has something to do with the pet.
